tox.ini

```diff
@@ -2,6 +2,8 @@
 envlist =
     docs, packaging, lint-py2, lint-py3,
     py27, py33, py34, py35, py36, py37, pypy
+_pip = python {toxinidir}/.tox_helpers/runpip.py
```

Comment:
But `_pip` is used for other environments too. How about a dedicated `[helpers]` section then?